tests/resources/try_compile/conanfile.txt

```diff
@@ -0,0 +1,5 @@
+[requires]
+fmt/10.1.1
```

Comment:
It is recommended to start using explicitly ``CMakeDeps`` generator in the conanfile.txt too.

Reply:
Done!